create-meta/java/src/main/java/io/cucumber/createmeta/CreateMeta.java

```diff
@@ -57,6 +57,17 @@
                 .build();
     }
 
+    public static String cleanSensitiveInformation(String value) {
```

Comment:
The only field that can contain credentials is the git remote, which is a URL. I think it would be safer to parse the value into a URL object, remove the credentials using the URL API and then write it out again.

Reply:
I've done this now